### DISCUSSION TITLE
top 'display bar' as in the demo

### DISCUSSION THREAD

**Discussion by sectorzero** [kovidgoyal/kitty#4469] [Q&A]
Apologies, if I don't know the terminology. I recently discovered Kitty (coming from Iterm2) and have been enjoying using the fast experience. But I haven't been able to figure one thing even after a bit of searching around. In the demo of Kitty, there is a nice top 'display bar'. How do I get it. Can you give some pointers and some advanced examples if possible. This has been bothering me a bit as I want something like this. If the author does not mind providing the config or how to achive the exact, it will be great. Atleast I am looking for a direction to go achieve that either way.

<img width="677" alt="Screen Shot 2022-01-07 at 2 10 57 PM" src="https://user-images.githubusercontent.com/892786/148614007-a9042b5d-539d-41df-bf33-a269e4263f6f.png">

Thanks in advance.

**Comment by page-down** [comment]
Yes, what you see is part of the kitty. Please refer to the following feature.

https://sw.kovidgoyal.net/kitty/kittens/panel/

**Comment by zzhaolei** [comment]
I think we can also look at this #4447

**Comment by kovidgoyal** [comment]
It uses the panel kitten, however, that is X11 specific, so it wont work
on macOS, which I assume you are using, as you mention iTerm

**Reply by arpangreat** [reply]
but specifically how to get the demo one working. Any config file provided ? :)

**Comment by kovidgoyal** [comment]
On Sun, Jan 30, 2022 at 11:03:24AM -0800, Swastik Acharyya wrote:
> but specifically how to get the demo one working. Any config file provided ? :)


Its not a config file. Its a program that runs and outputs to the kitty
panel. The program in question is one I wrote specifically for
myself, its not suitable for general use.
